stacks/lemp/setup.sh

```diff
@@ -5,37 +5,6 @@ set -euo pipefail
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
 apt-get install -y nginx php5-fpm php5-mysql php5-cli php5-curl git php5-dev mysql-server
-rm -f /etc/nginx/sites-enabled/default
-cat > /etc/nginx/sites-available/sandstorm-php <<EOF
-server {
-    listen 8000 default_server;
-    listen [::]:8000 default_server ipv6only=on;
-
-    # Allow arbitrarily large bodies - Sandstorm can handle them, and requests
-    # are authenticated already, so there's no reason for apps to add additional
-    # limits by default.
-    client_max_body_size 0;
-
-    # Prevent nginx from adding compression; this interacts badly with Sandstorm
-    # WebSession due to https://github.com/sandstorm-io/sandstorm/issues/289
-    gzip off;
-
-    server_name localhost;
-    root /opt/app;
-    location / {
-        index index.php;
-        try_files \$uri \$uri/ =404;
-    }
-    location ~ \\.php\$ {
```

Comment:
People frequently run into syntax error problems in the shell heredoc stuff, so I'm really excited we're getting rid of them!